Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs

```diff
@@ -578,12 +578,13 @@ private Size UpdateMessageSizing(IWin32Window? showOwner)
             {
                 // Find size of the label, with a max of 2/3 screen width
                 Screen? screen = showOwner != null ? Screen.FromHandle(showOwner.Handle) : Screen.PrimaryScreen;
-                SizeF scaledMonitorSize = screen.Bounds.Size;
-                scaledMonitorSize.Width *= 2 / 3.0f;
-                scaledMonitorSize.Height *= 0.95f;
-                SizeF messageSize = g.MeasureString(_text, _messageText.Font, scaledMonitorSize);
+                Size scaledMonitorSize = screen.Bounds.Size;
+                scaledMonitorSize.Width =(int)(scaledMonitorSize.Width * 2 / 3.0f);
+                scaledMonitorSize.Height = (int)(scaledMonitorSize.Height * 0.95f);
+                Font textFont = _messageText.StateCommon.Content.GetContentShortTextFont(PaletteState.Normal);
+                SizeF messageSize = TextRenderer.MeasureText(_text, textFont, scaledMonitorSize);
                 // SKC: Don't forget to add the TextExtra into the calculation
-                SizeF captionSize = g.MeasureString($@"{_caption} {TextExtra}", _messageText.Font, scaledMonitorSize);
+                SizeF captionSize = TextRenderer.MeasureText($@"{_caption} {TextExtra}", SystemFonts.CaptionFont, scaledMonitorSize);
```

Comment:
Should be using the font currently being used by the Theme

Reply:
> Hi @Smurf-IV
> 
> Oh, I missed that the caption is drawn with a custom font. I'm not sure how to get it though. I checked under `.StateCommon` but couldn't find anything that refers to the styling of the caption. Could you demonstrate how to do that? Or at least point me to the part of the code that paints the NC area? I checked the `KryptonForm.WindowChromePaint` method but I'm not sure how to go further.
> 
> I suspect that `VisualForm.Palette.BaseFont` is the right one but I don't know how to prove it.

This should work for (V90 & V85) KMBs: `KryptonManager.CurrentGlobalPalette.BaseFont`

Reply:
@Smurf-IV

Done.
Please note I'm only submitting for V85 for now. I'll see if I can update to the Preview version of VS to be able to build V90 too.